coopr-docs/docs/source/rest/node-usage.rst

```diff
@@ -0,0 +1,174 @@
+..
+   Copyright © 2012-2015 Cask Data, Inc.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+ 
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+
+:orphan:
+
+.. index::
+   single: Metrics 
+
+=======
+Metrics
+=======
+
+.. include:: /rest/rest-links.rst
+
+Admins are able to get metrics, such as the number of tasks that are in progress and queued as well as node hour usage.
+
+.. _metrics-node-usage:
+
```

Comment:
place two lines before the target and one line after. helps to show that the target applies to the next item and breaks up the code.